docs/units-and-global-variables.rst

```diff
@@ -186,6 +186,11 @@ Mathematical and Cryptographic Functions
 
     For further details, read `example usage <https://ethereum.stackexchange.com/q/1777/222>`_.
 
+.. note::
+
+    Since `EIP-2 <http://eips.ethereum.org/EIPS/eip-2#specification>`_, all transaction signatures with an ``s`` value
```

Comment:
I think this should be merged with the following warning. Please also clarify that EIP2 only applies to transaction signatures and not to the function/precompile ecrecover.

Reply:
Good idea, I think the last commit covers this.

Reply:
Looks good @chriseth